<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSetName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineAge">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>37</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSchool">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>63</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineMarks">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>89</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSetAge">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Age</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSetSchool">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set School</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonGetAge">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Age</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSetMarks">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Marks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonGetMarks">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Marks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSetAll">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonGetAll">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonQuit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>11</y>
     <width>27</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAge">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>37</y>
     <width>19</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSchool">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>64</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>School</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMarks">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>90</y>
     <width>28</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Marks</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonGetAge</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>getAge()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSetAll</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>setAllInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonGetMarks</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>getMarks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonQuit</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSetAge</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>setAge()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonGetAll</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>getAllInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSetMarks</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>setMarks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSetName</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>setName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSetSchool</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>setSchool()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setName()</slot>
  <slot>setAge()</slot>
  <slot>setSchool()</slot>
  <slot>setMarks()</slot>
  <slot>setAllInfo()</slot>
  <slot>getAge()</slot>
  <slot>getMarks()</slot>
  <slot>getAllInfo()</slot>
 </slots>
</ui>
